### ISSUE TITLE
About rl_loss

### ISSUE THREAD

**Issue by xcfcode** [ymfa/seq2seq-summarizer#7]
Thank you for this excellent job, I still have some questions about rl_loss, `rl_loss = neg_reward * sample_out.loss`, the `neg_reward` is obtained by `greedy_rouge - sample_rouge`, and the `sample_out.loss` means the cross-entropy loss, it is equal to `-LogP()`. However, in the paper,  self-critical policy gradient training algorithm uses `LogP()`, this confused me, could you please explain this?

**Comment by xcfcode** [comment]
I think I have already solved it, Thank you! you are right.
